src/_styles/home.less

```diff
@@ -88,10 +88,17 @@ body  {
   background-image: url(/images/denver-kitchen-remodel.jpg);
   background-repeat: no-repeat;
   color: #cd2026;
+  text-align: center;
```

Comment:
I wouldn't add `text-align: center` on the whole body, since this will make _every_ element have that same property